lib/IDE/SyntaxModel.cpp

```diff
@@ -458,10 +458,7 @@ std::pair<bool, Expr *> ModelASTWalker::walkToExprPre(Expr *E) {
   if (isVisitedBeforeInIfConfig(E))
     return {false, E};
 
-  if (E->isImplicit())
-    return { true, E };
-
-  if (auto *ParentTupleExpr = dyn_cast_or_null<TupleExpr>(Parent.getAsExpr())) {
+  auto addCallArgExpr = [&](const Expr *Elem, TupleExpr *ParentTupleExpr) {
     if (isCurrentCallArgExpr(ParentTupleExpr)) {
       CharSourceRange NR = parameterNameRangeOfCallArg(ParentTupleExpr, E);
```

Comment:
Can we use `Elem` here instead of `E`? If so, we can remove capture `[&]`.

Reply:
Yes, that was an oversight from my part. I couldn't remove the capture though, because I need to capture `this` (at least that was the compiler told me - I don't have much experience with C++)